app/templates/frameworks/_framework_actions.html

```diff
@@ -0,0 +1,110 @@
+{% if framework.status in ['standstill', 'live'] and application_made and not countersigned_agreement_file %}
+<li class="browse-list-item">
+  <a class="browse-list-item-link" href="{{ url_for('.download_agreement_file', framework_slug=framework.slug, document_name='result-letter.pdf') }}" download>
+    <span>Download your application {% if supplier_is_on_framework %}award{% else %}result{% endif %} letter (.pdf)</span>
+  </a>
+  {% if supplier_is_on_framework %}
+    <p>This letter is a record of your successful {{ framework.name }} application.</p>
+  {% else %}
+    <p>This letter informs you if your {{ framework.name }} application has been successful.</p>
```

Comment:
Yeah, @ralph-hawkins had a quote about not breaking up with people over text message.

Deliver bad news in a serious way, basically; don't just have a cheesy on-screen message, but let the official letter be the serious thing.

Also, Ralph, Roz and Cath have all looked at it and they're happy.